src/java.base/share/classes/java/net/SocketImpl.java

```diff
@@ -82,8 +82,12 @@ public SocketImpl() { }
      *
      * @apiNote
      * The {@link Socket} constructors to create a datagram socket
-     * are deprecated for removal. This method will be re-specified
-     * in a future release to not support creating datagram sockets.
+     * are deprecated for removal and have been respecified to throw
```

Comment:
This seems to talk about past, current and future behavior.

I thought we try to keep specifications focused on current behavior, with the exception of deprecation warnings.

Would it be possible to reword this without mentioning the past, avoiding the “have been respecified” part?

Interested users can always use release notes to observe history..?

Reply:
Hello Eirik,
> Would it be possible to reword this without mentioning the past, avoiding the “have been respecified” part?
that's a good point

Reply:
Hello Eirik,

> This seems to talk about past, current and future behavior.
> ...
> Would it be possible to reword this without mentioning the past, avoiding the “have been respecified” part?

That's a good point. I've now updated the PR to reword this. Hopefully that's better.